app/src/main/resources/config/application.yml

```diff
@@ -42,6 +46,7 @@ spring:
 graphql:
     packages:
         - "com.ritense.portal"
```

Comment:
Is this package still needed?

Reply:
Yes, a few modules are not renamed yet to nl.nl-portal. I don't want to touch these all in one PR.